src/main/webapp/WEB-INF/tags/instructor/search/feedbackResponse.tag

```diff
@@ -28,11 +28,9 @@
   <td>
     <ul class="list-group comments" id="responseCommentTable-${fsIndx}-${qnIndx}-${responseIndex}"
         <c:if test="${empty responseRow.feedbackResponseComments}">style="display:none"</c:if>>
-
-      <c:forEach items="${responseRow.feedbackResponseComments}" var="frc" varStatus="i">
-        <shared:feedbackResponseCommentRow frc="${frc}" firstIndex="${fsIndx}" secondIndex="${qnIndx}" thirdIndex="${responseIndex}" frcIndex="${i.count}" />
-      </c:forEach>
-
+            <c:forEach items="${responseRow.feedbackResponseComments}" var="frc" varStatus="i">
+                <shared:feedbackResponseCommentRow frc="${frc}" firstIndex="${fsIndx}" secondIndex="${qnIndx}" thirdIndex="${responseIndex}" frcIndex="${i.count}"/>
+            </c:forEach>
```

Comment:
Wrong indentation here: use the original indentation. As this file has already been updated to 2 spaces, we should continue to use 2 spaces indentation.